tests/measurement/hooks/test_addtask_hook.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2018 by Exopy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Test the AddTask post hook.
+
+"""
+import pytest
+from atom.api import Tuple
+from exopy.tasks.api import RootTask, SimpleTask
+from enaml.workbench.api import Workbench
+from exopy.testing.measurement.dummies import DummyEngine
+from exopy.tasks.tasks.base_views import RootTaskView
+
+from exopy.testing.util import show_widget
+pytest_plugins = str('exopy.testing.tasks.fixtures')
+
+
+@pytest.fixture
+def addtaskhook(measurement):
+    """Create an AddTaskHook.
+
+    """
+    hook = measurement.plugin.create('post-hook', 'exopy.addtask_hook')
+    return hook
+
+
+@pytest.fixture
+def addtaskview(measurement_workbench, addtaskhook):
+    """Create an AddTaskView.
+
+    """
+    # on est obligé d'aller chercher dans le vrai manifest de measurement pour
+    # trouver le make_view
+    meas = measurement_workbench.get_plugin('exopy.measurement')
+    decl = meas.get_declarations('exopy.addtask_hook') # check how to get the declaration
+    view = decl.make_view(measurement_workbench, addtaskhook)
+    return view
+
+
+def test_new_addtask_hook(addtaskhook):
+    """Testing the creation of an AddTask post-hook
+
+    """
+    assert type(addtaskhook.root_task) == RootTask
+    assert type(addtaskhook.workbench) == Workbench
+    assert type(addtaskhook.dependencies) == Tuple
```

Comment:
You are mixing the type of the decorator and the type of the value it returns. It should be tuple.